adr/00011-python-containerisation.md

```diff
@@ -0,0 +1,21 @@
+# Python application building and deployment using containers
+
+Date: 2019-01-30
+
+## Status
+
+Proposed
+
+## Context 
+
+Python and Python applications are a mess to install and run. 
```

Comment:
I'd probably prefer 'complex' rather than 'mess' ... and also probably worth mentioning the propensity to sully your development environment too.

Reply:
everyone's a critic ;) I'll use [complicated](https://www.infoq.com/presentations/Simple-Made-Easy)